api/demos/demo-cohort.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "demo-cohort",
+  "description": "demo",
+  "criteria": {"includes":[{"id":"includes_twr4se71q","items":[{"id":"items_s64k36qwq","type":"ICD9","searchParameters":[{"parameterId":"param6","name":"Cholera","value":"001","type":"ICD9","subtype":"","group":true,"domain":""}],"modifiers":[]}]}],"excludes":[]}
```

Comment:
If we're expecting people to hand-modify this, should format this JSON onto more than one line.

Reply:
I can do that. For now this is also mostly a placeholder until we get real demo cohorts... (And maybe an export as JSON functionality for a cohort definition?)